module/localisation/RobotParticleLocalisation/tests/Localisation FlowChart

```diff
@@ -0,0 +1 @@
+<mxfile host="app.diagrams.net" modified="2021-03-18T04:23:32.569Z" agent="5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.141 Safari/537.36" etag="WNySBxAFqS_fUB_epxsu" version="14.4.9" type="device"><diagram id="SexmEN1cIrFT9L4E9Oxq" name="Page-1">7Vxbc5s4FP41mdl9qIerwI+5tZ2dtE2bzW53XzoyyFgbQIyQE3t//UrcDEi2SYwhbjZ9qHUsMNY533euyZl5Ga0+UJgsPhEfhWeG5q/OzKszw9B1x+X/Cck6lwDLzAUBxX6xaSO4w/+iQqgV0iX2UdrYyAgJGU6aQo/EMfJYQwYpJU/NbXMSNj81gQGSBHceDGXpn9hni1zqGs5G/hHhYFF+sg6m+TsRLDcX3yRdQJ881UTm9Zl5SQlh+atodYlCcXjlueTXvd/ybvVgFMWsywXTB+3r/dNv5td3QapfW3+fz78H70ptpGxdfmPk8wMoloSyBQlIDMPrjfSCkmXsI3Fbja82e24ISbhQ58J/EGPrQptwyQgXLVgUFu/yJ6br78X12eKv+uJq1VgJyzHscnmLKI4QQ7TYkz+/eOit51KIUrKkHtpxGKV9QRogtmOfUWmPmz0i/Gnoml9HUQgZfmw+ByzsL6j2FZeeUwrXtQ0JwTFLa3e+FQK+oYCSYRZ2VABJBy1179nv2vXt/EX+AOWq9k02osyCnmFNxXd+hOGyOIU7fphMNrEw5PgVpvS0wAzdJTDTyxOnkKahzHEYXpKQ0Ow6c+56yPO4PGWUPKDaOzPXtuyNOTwiytBqt0HICiwv0KbNk7byo9OeagRQHO6ihv3ywFUqr53784/VdkcB6QqzDKOTqQGKtYCpPtHMabHeIFUs1rVFG6YvgLw1AOSNjpC3nT4wL4HUmTZBWtlVeYv8wYqrNmb0XPJof44JrOOzgSGxAYez0KIBQn7WFzPxKhCv5pREfOc3MiPslhMG9kJ0Q7gTxik/XRJPFkkiQYD700S89JazDkwyy+FwM6sE0HsIMpB8WbIQx6iQ+5A+fOG3wSwz54lmN4VGJhU70xw5uqYgqjkCaqLynelM64mobBc0tFp4hDpPVdFHnajAsYjK0iQtDUhUeo2ktIm9m6LUdDSxdxBSz9RjdqQec4sRdKaew1RqjxogvuMA1OyWjoCzR0vH9hpdVVeL8weMFG2t5VTM3ZFie7/uDOAcTMk5XJJ4joMlzRj/BT7CU/iIpiE+L9j0beT6lorDXWNmAtAPh1dkXCpLU5C4OSiJOydO4rtiyiHYQbe60oM9JrOXj1mD4DeUIlaCjP//cZ0QtuDC9O3i0ZWTv4HxCMbAY1c89J8S6W1v1DrYHH9SSiTfqKVJ3TCaN+ort2p/jqO1dH6M3EqTwBuJ2uskocRDafqZ4BRdYZgpXQZvvvfz/acf3758uL++27aDCkL4BmNuBVvTL0j9/VgucyWgyJVs8U+Fa5D9iCtIzGry/KenHKq0iqrYI/tfvcyzBsG7rUtn/Vr97846z8SZVqF8o240YuBu2F1d85ieuXzKGrg5DH+kDDL0y689IVFVtRgXiW3PO+1WdrWOBkTrZID4OgNhwz1KhNA2FEvrVjTty/eCUQi6d710YEFwaP3iMBp0JRr8nZsql9wnPqfCt5OT2M2GlOmoagT6gDGKOUqht08ITLsGAody2GEQmEoQ2Or+o1UgRjAmsySKJ4WwfnrCBDHHwg2cofCWpDirs5lXHCuMw8a8CMUbF1VbpNHOED+1e5yHOBDXMqG5C1isPK4G4UcukppPoQsSzZZpzc3UWi/8/YAj+UmkRxck78NcVpMc2kaYhTfMW+A4EGhbRzMivhfj96SKQGaYPrHV9EOmIwcstiJgsY+GSrn/XnFkSZHXEWalkH9IJRe0mSV7gjvLug9Ot1pbylDS0rQQ1dW8l0pXqBz4UVErRO5cqUPguWg27ynobFUbDF1R7tEVSjxas9+UK+PPUWJZRd+esP8kmms7RdWUhtIpHk9zctome8nYPxdTaYLPQpim2DvrMhm1ty/5kqyhR3fadXIqN25Zpx0Z88DEwTJacZRtN2/RtbRoOa0baa0bHTkDMadjhF+n2mfvmu1ssc1hIj1TTnZeAZUPE0i5+32wPqgPthTE/T++th1WV/LPG46jDbLIwfHrCniHGm5teS5Vu2PQ4dbT8WW72h16E3/GgAC0nY4A7Kce++xxpNYgurlncF3aP8TkenmGNXbgZymaLe0aIxdfkkdIMYy9rBsDYCSAHs/SpNLrod0ZV0EPL2/OzEOcfDyr19AOohBguU0FKVqmU0XH9GiNmrLM+cKU+T1GoT8e7w9U6Gg3TUwF8Q8aZIHTaa/t63PXuV+0ucG+bP3IjTfQeUB1zIgMyJUuUc99Ox1u05UxOGiHG5z6qOeQeQ7oOjsCwKiokquQf+AUk/gDgeFPn96AZmGuKu/XvZzqV2KO5uXKVu3JQmwHwAb5pVwAusJu1E6tM87U3qnrbDoqVQKJKj8hmC4pilCWkb69OZOJ3cwSFDN4w06aOKdTilVkCX1i6jjDdPKU/MDDdHLf420lAdbYSYAzzp8AeYUIO9QZvVKEyTNkUe7l3g7IbEW1qyeQ8eXmb/jkOtv8JSTz+j8=</diagram></mxfile>
```

Comment:
What is this?

Reply:
@danielginn will have to answer that

Reply:
It looks like it should be some sort of documentation like thing. In which case it either needs to be in NUbook, or in the docs folder.

You can open it at [app.diagrams.net](https://app.diagrams.net).

![image](https://user-images.githubusercontent.com/2320984/112108555-770c8880-8c04-11eb-9aa1-e4eef409649d.png)

Reply:
I'll move it to the docs folder then :+1: 

@danielginn if you have other things you wanted from this diagram, or you don't need it any more drop me a line